CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# v17.0.0-snapshot
+
+Kubernetes API Version: 1.17.13
+
+**Important Information:**
+
+- The Kubernetes Python client versioning scheme has changed. The version numbers used till Kubernetes Python Client v12.y.z lagged behind the actual Kubernetes minor version numbers. From this release, the client is moving a version format `vY.Z.P` where `Y` and `Z` are respectively from the Kubernetes version `v1.Y.Z` and `P` would incremented due to changes on the Python client side itself. Ref: https://github.com/kubernetes-client/python/issues/1244
```

Comment:
We should add this to README. Maybe when we do the first v17 alpha release

Reply:
Yes. That's what my thoughts were. After we cut the first Alpha, we should put both the things in the Important Imformation block in the README.